xen/arch/x86/tboot.c

```diff
@@ -419,8 +419,10 @@ int __init tboot_protect_mem_regions(void)
 {
     int rc;
 
+    /*
     if ( !tboot_in_measured_env() )
```

Comment:
```suggestion
    if ( !tboot_in_measured_env() && sl_status == 0 )
```
Plus `extern uint32_t sl_status;` earlier if needed, and don't comment out this conditional.